pkg/minishift/systemtray/minitray.go

```diff
@@ -0,0 +1,106 @@
+/*
+Copyright (C) 2017 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package systemtray
+
+import (
+	"fmt"
+	goos "os"
+	"os/exec"
+	"runtime"
+	"syscall"
+
+	"github.com/golang/glog"
+	minishiftConfig "github.com/minishift/minishift/pkg/minishift/config"
+	"github.com/minishift/minishift/pkg/util/os"
+	"github.com/minishift/minishift/pkg/util/os/process"
+)
+
+type MiniTray struct {
+	globalConfig *minishiftConfig.GlobalConfigType
+}
+
+func NewMiniTray(globalConfig *minishiftConfig.GlobalConfigType) *MiniTray {
+	return &MiniTray{globalConfig: globalConfig}
+}
+
+func systemTrayCommand() (*exec.Cmd, error) {
+	cmd, err := os.CurrentExecutable()
+	if err != nil {
+		return nil, err
+	}
+	args := []string{
+		"systemtray",
+	}
+	exportCmd := exec.Command(cmd, args...)
+	// don't inherit any file handles
+	exportCmd.Stderr = nil
+	exportCmd.Stdin = nil
+	exportCmd.Stdout = nil
+	exportCmd.SysProcAttr = process.SysProcForBackgroundProcess()
+	exportCmd.Env = process.EnvForBackgroundProcess()
+	fmt.Println("Finished setting up the command")
+	return exportCmd, nil
+}
+
+func (s *MiniTray) EnsureRunning() error {
+	fmt.Println("inside ensure running")
```

Comment:
these are log type output... not suitable to use `fmt.Println`

Reply:
Left over from debugging printlns.. this will be removed